src/tools/sssctl/sssctl_config.c

```diff
@@ -64,6 +64,9 @@ errno_t sssctl_config_check(struct sss_cmdline *cmdline,
     /* Open config file */
     ret = sss_ini_config_file_open(init_data, SSSD_CONFIG_FILE);
     if (ret != EOK) {
+        if (ret == ENOENT) {
```

Comment:
I tend to agree with @lslebodn on his comments.

And yes, I do believe that mentioning the path to sssd.conf is desired.